### PR TITLE
#1864 - when updating from nuget, remove NuGetPackageImportStamp

### PR DESCRIPTION
**Pull request by 0x53A**
ref https://github.com/fsprojects/Paket/issues/1864

### CODE REVIEW

**Comment by forki** [comment]
can you please rebase this?

**Comment by 0x53A** [comment]
done

File: src/Paket.Core/ProjectFile.fs [hunk 1]
```diff
@@ -183,6 +184,12 @@ module ProjectFile =
         use stream = fileInfo.OpenRead()
         loadFromStream fileInfo.FullName stream
 
+    let loadFromString (fullName:string) (text:string) =
```
Comment (forki) [inline src/Paket.Core/ProjectFile.fs]: why is this needed?

**Comment by forki** [comment]
cool stuff. thanks
